docs/testbed/README.testbed.Sentinel.md

```diff
@@ -0,0 +1,66 @@
+# Sentinel
+
+Currently, we don't have a system to monitor all devices of the testbed, including SONiC DUT, EOS neighbors and PTF. Testbed management is a web tool for managing testbeds, developers use it to book testbed for development work, it also allocates corresponding testbeds for nightly test. 
```

Comment:
It would be better if the sentinel also can monitor health status of test server.

Reply:
Sure, it's very easy to add this part.